### PR TITLE
feat: denoise

### PR DESCRIPTION
**Pull request by coryhh**

### CODE REVIEW

File: arex-schedule-web-api/src/main/java/com/arextest/schedule/beans/ExecutorServiceConfiguration.java [hunk 1]
```diff
@@ -17,69 +19,62 @@
...
-            .build();
-        return new ThreadPoolExecutor(CORE_POOL_SIZE, CORE_POOL_SIZE, KEEP_ALIVE_TIME,
-            TimeUnit.MILLISECONDS,
-            new LinkedBlockingQueue<>(COMPARE_QUEUE_MAX_CAPACITY_SIZE),
-            threadFactory,
+            .setDaemon(true).setUncaughtExceptionHandler(this).build();
+        return new ThreadPoolExecutor(CORE_POOL_SIZE, CORE_POOL_SIZE, KEEP_ALIVE_TIME, TimeUnit.MILLISECONDS,
+            new LinkedBlockingQueue<>(COMPARE_QUEUE_MAX_CAPACITY_SIZE), threadFactory,
+            new ThreadPoolExecutor.CallerRunsPolicy());
+    }
```
Comment (wildeslam) [inline arex-schedule-web-api/src/main/java/com/arextest/schedule/beans/ExecutorServiceConfiguration.java:70]: Rollback structural diffs.

Comment (coryhh) [reply arex-schedule-web-api/src/main/java/com/arextest/schedule/beans/ExecutorServiceConfiguration.java:70]: This time, the code format style is ignored. In future submissions, 
[https://github.com/arextest/arex-api/blob/main/intellij-java-google-style.xml]() will be used uniformly.

**Review by QizhengMo — APPROVED** [review]

File: arex-schedule-web-api/src/main/java/com/arextest/schedule/model/ReplayActionCaseItem.java [hunk 2]
```diff
@@ -56,6 +58,12 @@ public class ReplayActionCaseItem {
      */
     private long executionStartMillis;
 
+    /**
+     * the compare mode of the case, {@link CompareModeType}
+     */
+    @JsonIgnore
+    private CompareModeType compareMode = CompareModeType.QUiCK;
```
Comment (wildeslam) [inline arex-schedule-web-api/src/main/java/com/arextest/schedule/model/ReplayActionCaseItem.java:65]: unused?

File: arex-schedule-web-api/src/main/java/com/arextest/schedule/planexecution/impl/DefaultExecutionContextProvider.java [hunk 3]
```diff
@@ -34,12 +29,13 @@
...
 @AllArgsConstructor
-public class DefaultExecutionContextProvider implements PlanExecutionContextProvider<DefaultExecutionContextProvider.ContextDependenciesHolder> {
+public class DefaultExecutionContextProvider
+    implements PlanExecutionContextProvider<DefaultExecutionContextProvider.ContextDependenciesHolder> {
     private final ReplayActionCaseItemRepository replayActionCaseItemRepository;
     private final ReplaySenderFactory replaySenderFactory;
 
     private static final RetryTemplate RETRY_TEMPLATE = RetryTemplate.builder().maxAttempts(3).build();
-    private static final String CONTEXT_PREFIX = "batch-";
+    // private static final String CONTEXT_PREFIX = "batch-";
```
Comment (wildeslam) [inline arex-schedule-web-api/src/main/java/com/arextest/schedule/planexecution/impl/DefaultExecutionContextProvider.java:38]: remove unused code

File: arex-schedule-web-api/src/main/java/com/arextest/schedule/bizlog/BizLogger.java [hunk 4]
```diff
@@ -1,217 +1,202 @@
...
+    }
+    // endregion
+
     public enum BizLogContent {
         PLAN_START(0, "Plan passes validation, starts execution."),
         PLAN_CASE_SAVED(1, "Plan saved total {0} cases to send, took {1} ms."),
         PLAN_CONTEXT_BUILT(2, "{0} execution context built, took {1} ms."),
-        PLAN_DONE(3, "Plan send job done normally."),
-        PLAN_ASYNC_RUN_START(4, "Plan async task init."),
+        PLAN_DONE(3, "Plan send job done normally."), PLAN_ASYNC_RUN_START(4, "Plan async task init."),
```
[the remark this answers is not in the document]
Comment (coryhh) [reply arex-schedule-web-api/src/main/java/com/arextest/schedule/bizlog/BizLogger.java]: done

**Review by wildeslam — APPROVED** [review]

**Review by binyu1005 — APPROVED** [review]

**Pull request merged**